stratum/hal/lib/tdi/es2k/es2k_extern_manager_test.cc

```diff
@@ -0,0 +1,353 @@
+// Copyright 2023-2024 Intel Corporation
```

Comment:
2024 only

Reply:
Some of the IP content is adapted from sources that were created earlier than this year.